app/views/help/cookie_settings.html.erb

```diff
@@ -44,13 +45,15 @@
 
       <% cookies_usage_hint = capture do %>
         <p class="govuk-body govuk-hint"><%= t('cookies.usage_info') %></p>
-        <p class="govuk-body govuk-hint"><%= t('cookies.google_info') %>:</p>
-        <ul class="govuk-list govuk-list--bullet govuk-hint">
+        <p class="govuk-body govuk-hint"><%= t('cookies.google_info') %></p>
+        <ul class="govuk-list-body govuk-list--bullet govuk-hint">
```

Comment:
Just wondering - what's the `govuk-list-body` class doing?

Reply:
@injms It's ensuring the same text style as `govuk-body` while allowing the list styling. Previously the text was much harsher in colour than the rest of the page.

Reply:
Thanks for the explanation. I asked as there aren't any styles associated with the `govuk-list-body` class - so you'll get the same effect by removing the class:

```suggestion
        <ul class="govuk-list--bullet govuk-hint">
```

Reply:
@injms Sorry for being late to this - you're correct, and will apply the change shortly.